gradle.properties

```diff
@@ -13,3 +13,5 @@ org.gradle.jvmargs=-Xmx8g
 # org.gradle.parallel=true
 android.enableJetifier=true
 android.useAndroidX=true
+android.jetifier.ignorelist=bcprov-jdk15on
```

Comment:
what's the difference btw ignoreList  and blackList?

Reply:
One is probably enough. I wanted to be really sure :-) I will find out if one (or both) can be removed.

Reply:
Ignorelist does nothing /ATM) and blacklist is necessary to prevent the following error: 
```
Execution failed for task ':app:kaptGenerateStubsDebugUnitTestKotlin'.
> Could not resolve all artifacts for configuration ':app:debugUnitTestCompileClasspath'.
   > Failed to transform bcprov-jdk15on-1.68.jar (org.bouncycastle:bcprov-jdk15on:1.68) to match attributes {artifactType=android-classes-jar, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-api}.
      > Execution failed for JetifyTransform: /Users/biafra/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcprov-jdk15on/1.68/46a080368d38b428d237a59458f9bc915222894d/bcprov-jdk15on-1.68.jar.
         > Failed to transform '/Users/biafra/.gradle/caches/modules-2/files-2.1/org.bouncycastle/bcprov-jdk15on/1.68/46a080368d38b428d237a59458f9bc915222894d/bcprov-jdk15on-1.68.jar' using Jetifier. Reason: IllegalArgumentException, message: Unsupported class file major version 59. (Run with --stacktrace for more details.)
           Suggestions:
            - Check out existing issues at https://issuetracker.google.com/issues?q=componentid:460323&s=modified_time:desc, it's possible that this issue has already been filed there.
            - If this issue has not been filed, please report it at https://issuetracker.google.com/issues/new?component=460323 (run with --stacktrace and provide a stack trace if possible).
```